### ISSUE TITLE
Are the "don't"s needed here?

### ISSUE THREAD

**Issue by toraritte** [rzach/forallx-yyc#102]
https://github.com/rzach/forallx-yyc/blob/730c38819ceb25485b894b1f8f9cfdaed45317b4/forallx-yyc-what.tex#L203

The changed text below is what sounds correct to me, but I've been struggling with math and logic long enough that it may be just me not understanding something fundamental...

> We might call an argument \define{nomologically valid} if there are no counterexamples that <s>don't</s> violate the laws of nature, and an argument \define{conceptually valid} if there are no counterexamples that <s>don't</s> violate conceptual connections between words.

---

Thank you making these resources freely available and for putting your time and effort into it!

**Comment by rzach** [comment]
No, the double negation is right: any potential counterexample does violate the laws of nature/conceptual connections.

I'll leave this open: it might be worded better but I have to think about how.
